guides/v2.1/release-notes/ReleaseNotes2.1.16CE.md

```diff
@@ -0,0 +1,212 @@
+
+
+
+
+### Store
+
+<!--- ENGCOM-2627 -->*  The `$product->getUrlInStore()` method now returns more compact URLs. *Fix submitted by [Burlacu Vasilii](https://github.com/vasilii-b) in pull request [16310](https://github.com/magento/magento2/pull/16310)*. [GitHub-16273](https://github.com/magento/magento2/issues/16273)
+
+
+### Testing
+
+<!--- ENGCOM-2844 -->*  `\Magento\Cms\Model\Block` is now covered by unit tests. *Fix submitted by [Dmytro Cheshun](https://github.com/dmytro-ch) in pull request [17682](https://github.com/magento/magento2/pull/17682)*.
+
+<!--- ENGCOM-2889 -->*  The `\Magento\Catalog\Test\Unit\Cron\AvailabilityCheckerTest` and `\Magento\Catalog\Test\Unit\Cron\DeleteOutdatedPriceValuesTest` classes are now covered by unit tests. *Fix submitted by [Jignesh Baldha](https://github.com/jignesh-baldha) in pull request [17606](https://github.com/magento/magento2/pull/17606)*.
+
+<!--- ENGCOM-2909 -->*  The `\Magento\Search\Model\SynonymAnalyzer` and `\Magento\Search\Model\SynonymAnalyzer` classes are now covered by unit tests. *Fix submitted by [Zebra](https://github.com/furseyev) in pull request [17839](https://github.com/magento/magento2/pull/17839)*.
+
+<!--- ENGCOM-2928 -->*  `\Magento\Sales\Model\Validator` is now covered by unit tests. *Fix submitted by [Dmytro Cheshun](https://github.com/dmytro-ch) in pull request [17940](https://github.com/magento/magento2/pull/17940)*.
+
+
+
+
+### Theme
+
+<!--- ENGCOM-2917 -->* Magento now displays the wishlist icon on the shopping cart in mobile view. *Fix submitted by [Hitesh](https://github.com/hitesh-wagento) in pull request [17912](https://github.com/magento/magento2/pull/17912)*. [GitHub-17851](https://github.com/magento/magento2/issues/17851)
+
+
+
+
+
+### UI
+
+<!--- ENGCOM-2825 -->* The JavaScript validation rule used to validate AM/PM time settings now works as expected when JavaScript is minified. *Fix submitted by [Dmytro Cheshun](https://github.com/dmytro-ch) in pull request [17689](https://github.com/magento/magento2/pull/17689)*. [GitHub-17648](https://github.com/magento/magento2/issues/17648)
+
+<!--- ENGCOM-2835 -->* The message list component message type now has a message type of success. Previously, this type was always error when the parameters property was specified. *Fix submitted by [Dmytro Cheshun](https://github.com/dmytro-ch) in pull request [17702](https://github.com/magento/magento2/pull/17702)*. [GitHub-17700](https://github.com/magento/magento2/issues/17700)
+
+<!--- ENGCOM-2898 -->* The WYSIWYG editor now preserves the transparent background of .png  thumbnails. Previously, Magento rendered these transparent backgrounds black. *Fix submitted by [Eduard Chitoraga](https://github.com/eduard13) in pull request [17855](https://github.com/magento/magento2/pull/17855)*. [GitHub-14248](https://github.com/magento/magento2/issues/14248)
```

Comment:
This description seems to suggest that an image with a transparent bg displayed in the WYSIWYG editor has a black bg, but is fine otherwise. But if I understand the PR correctly, it's uploading an image with a transparant background which causes it to turn black wherever you display it, either on the frontend of the shop or in the WYSIWYG editor or someplace else. So this probably has nothing to do with the WYSIWYG editor itself but more with processing the image after uploading it.

It might also make sense to make a quick note that this only happens with the GD2 image library. Magento comes with support for both GD2 and ImageMagick, the problem was only with the GD2 library, people using the ImageMagick library didn't run into this problem.